apidoc/Titanium/Platform/Platform.yml

```diff
@@ -43,13 +43,38 @@ methods:
   - name: openURL
     summary: |
         Opens this URL using the system's default application for its protocol.
+    description: |
+        **iOS Note**: For iOS 10 and later, this method is performed asynchronously 
+        and will call the function passed as an (optional) third parameter instead
+        of returning a boolean synchronously.
+        
+        Example:
+          
+            Ti.Platform.openURL('myapp://', {
+              'UIApplicationOpenURLOptionUniversalLinksOnly': true
+            }, function(e) {
+              Ti.API.info('URL open successfully? ' + e.success);
+            });
+
     platforms: [android, iphone, ipad]
     returns:
         type: Boolean
     parameters:
       - name: url
         summary: The url to open.
         type: String
+      - name: options
+        summary: |
+            The optional options to pass to the URL handling (iOS 10+). Pass a
+            dictionary with one or more of the following string-keys:
+                * UIApplicationOpenURLOptionsSourceApplicationKey
+                * UIApplicationOpenURLOptionsAnnotationKey
+                * UIApplicationOpenURLOptionsOpenInPlaceKey
+                * UIApplicationOpenURLOptionUniversalLinksOnly
```

Comment:
We should give the detail about the values of these string-keys. e.g For UIApplicationOpenURLOptionUniversalLinksOnly value should be Bool or give an apple link here.
And normally we map apple keys with Titanium keys. Isn't it good to use titanium keys?

Reply:
The reason I've not made this a constant is because we cannot use constants as dictionary-keys, so the dev would need to declare a variable first and then use that one as a key. I'll add more infos about the constants though!